src/pages/Lectures.jsx

```diff
@@ -28,10 +28,14 @@ const Lectures = ({ course, lectures }) => {
       <Route exact path={path}>
         <div>
           <Title>{course}</Title>
-          <ProgressBar
-            percent={percent}
-            note={`${watched}/${total} Lectures Completed`}
-          />
+          {lectures.length === 0 ? (
+            <></>
+          ) : (
```

Comment:
```suggestion
          {lectures.length > 0  && (
```
don't need empty fragment